src/GR.jl

```diff
@@ -3647,64 +3647,10 @@ const gr3 = GR.GR3
 
 # Convenience functions
 include("jlgr.jl")
-
-colormap() = jlgr.colormap()
-figure(; kwargs...) = jlgr.figure(; kwargs...)
-gcf() = jlgr.gcf()
-hold(flag) = jlgr.hold(flag)
-usecolorscheme(index) = jlgr.usecolorscheme(index)
-subplot(m, n, p) = jlgr.subplot(m, n, p)
-plot(args...; kwargs...) = jlgr.plot(args...; kwargs...)
-oplot(args...; kwargs...) = jlgr.oplot(args...; kwargs...)
-semilogx(args...; kwargs...) = jlgr.plot(args...; kwargs..., xlog=true)
-semilogy(args...; kwargs...) = jlgr.plot(args...; kwargs..., ylog=true)
-loglog(args...; kwargs...) = jlgr.plot(args...; kwargs..., xlog=true, ylog=true)
-step(args...; kwargs...) = jlgr.step(args...; kwargs...)
-scatter(args...; kwargs...) = jlgr.scatter(args...; kwargs...)
-stem(args...; kwargs...) = jlgr.stem(args...; kwargs...)
-barplot(args...; kwargs...) = jlgr.barplot(args...; kwargs...)
-histogram(x; kwargs...) = jlgr.histogram(x; kwargs...)
-polarhistogram(x; kwargs...) = jlgr.polarhistogram(x; kwargs...)
-contour(args...; kwargs...) = jlgr.contour(args...; kwargs...)
-contourf(args...; kwargs...) = jlgr.contourf(args...; kwargs...)
-hexbin(args...; kwargs...) = jlgr.hexbin(args...; kwargs...)
-heatmap(D; kwargs...) = jlgr.heatmap(D; kwargs...)
-heatmap(x, y, z; kwargs...) = jlgr.heatmap(x, y, z; kwargs...)
-polarheatmap(D; kwargs...) = jlgr.polarheatmap(D; kwargs...)
-nonuniformpolarheatmap(x, y, z; kwargs...) = jlgr.nonuniformpolarheatmap(x, y, z; kwargs...)
-wireframe(args...; kwargs...) = jlgr.wireframe(args...; kwargs...)
-surface(args...; kwargs...) = jlgr.surface(args...; kwargs...)
-volume(V; kwargs...) = jlgr.volume(V; kwargs...)
-plot3(args...; kwargs...) = jlgr.plot3(args...; kwargs...)
-scatter3(args...; kwargs...) = jlgr.scatter3(args...; kwargs...)
-title(s) = jlgr.title(s)
-redraw(; kwargs...) = jlgr.redraw(; kwargs...)
-xlabel(s) = jlgr.xlabel(s)
-ylabel(s) = jlgr.ylabel(s)
-drawgrid(flag) = jlgr.drawgrid(flag)
-xticks(args...) = jlgr.xticks(args...)
-yticks(args...) = jlgr.yticks(args...)
-zticks(args...) = jlgr.zticks(args...)
-xticklabels(s) = jlgr.xticklabels(s)
-yticklabels(s) = jlgr.yticklabels(s)
-legend(args...; kwargs...) = jlgr.legend(args...; kwargs...)
-xlim(a) = jlgr.xlim(a)
-ylim(a) = jlgr.ylim(a)
-savefig(filename; kwargs...) = jlgr.savefig(filename; kwargs...)
-meshgrid(vx, vy) = jlgr.meshgrid(vx, vy)
-meshgrid(vx, vy, vz) = jlgr.meshgrid(vx, vy, vz)
-peaks(n...) = jlgr.peaks(n...)
-imshow(I; kwargs...) = jlgr.imshow(I; kwargs...)
-isosurface(V; kwargs...) = jlgr.isosurface(V; kwargs...)
-isosurface(V, v; kwargs...) = jlgr.isosurface(V, isovalue=v; kwargs...)
```

Comment:
This actually added a new definition for `isosurface`. I've moved that into `jlgr` itself.